.gitignore

```diff
@@ -168,3 +168,8 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+/samples/avcodec-59.dll
+/samples/nppig64_11.dll
+/samples/avutil-57.dll
+/samples/avformat-59.dll
+*.dll
```

Comment:
the last line should cover the other 4 cases

Reply:
resolved this in another rebase